app/javascript/techradar.js

```diff
@@ -137,7 +200,7 @@
                     })
                     .attr("r", 10)
                     .attr("class", function (d) {
-                        return (hasSpotMoved(d)) ? "spot hasMoved" : "spot";
+                        return ((hasSpotMoved(d)) ? "spot hasMoved" : "spot") + " " + quadrant.cssClass;
```

Comment:
Here I see the reason for the change in the `quadrant` data-structure - away from just a color towards a `cssClass`-attribute.
--but--
Unfortunately this change is really problematic since now we got a reference from within the data-structure (not even the code, but the bare data) into CSS!
In case someone is crafting that data through a Web-UI or via a REST-interface (what we actually do) he/she cannot simply change the color or introduce a new quadrant (without calling the IT-guy to define an additional CSS-class).
Having said that I would really suggest to switch back to simple color-information (which can be defined i.e. via a color-picker in the Web-UI).